### ISSUE TITLE
allow connecting accounts

### ISSUE THREAD

**Issue by snarfed** [snarfed/bridgy#38]
e.g. for the same person's accounts on each silo. could use indieauth to also let them attach their domain, but not strictly necessary.

**Comment by snarfed** [comment]
I don't really have any use cases in mind that connecting accounts like this would serve. Open to ideas though!

**Comment by snarfed** [comment]
Documented on https://brid.gy/about#connect

**Comment by davidcelis** [comment]
One thing I thought of about this since opening #1194; I don't know how Bridgy's crawling/polling is actually implemented, but for people who connect multiple social networks to a single website, it seems like Bridgy ends up crawling their website separately for each of those networks. Connecting accounts could ease the number of crawling operations Bridgy has to do, but that also seems like something you could solve independently if the amount of crawling ever becomes concerning to you

**Comment by snarfed** [comment]
True! Good point. Polling h-feeds for syndication links is a relatively small amount of Bridgy's total load, but you're right, we could consolidate it for web sites with multiple silo accounts. And you're also right that we could do that with or without the kind of UI connection described in this issue.
